autosklearn/evaluation/abstract_evaluator.py

```diff
@@ -361,23 +370,22 @@ def file_output(
                     },
                 )
 
+        # Abort if we don't want to output anything
+        # note: since disable_file_output can be a list, we have to explicitly
+        # compare it against True
+        if self.disable_file_output == True:
+            return None, {}
+
         num_run = str(self.num_run)
+        write_everything = not isinstance(self.disable_file_output, list)
```

Comment:
I think that `not isinstance(self.disable_file_output, list)` is the same as `self.disable_file_output==False`. Because at this point in the code we already ruled out that `self.disable_file_output` is not True. 
I'm not quite sure which way to put it is better.